index.html

```diff
@@ -665,6 +666,13 @@ <h2 class="ui header">Separate glyphs <em class="fontname" style="display: none;
     })
   })
 
+  $('#reset').click(function () {
+    $('#glyphs').empty()
+    $('#source').val('')
+    $('#createFont').prop("disabled", false)
+    $('#extract').prop("disabled", false)
```

Comment:
Good idea to implement a reset button!

Here maybe the export button should be disabled again?

Reply:
It's still a work in progress ;) 
I was battling git to be able to push — more commits are coming in the course of the day (and the week-end)

Reply:
Also the "add"-button...

Reply:
Okay, thank you. Don't feel pressured, it's a hobby for both of us.

Reply:
yep. I'm on it